neevaco/neeva-android
# Add missing Intent filter for neeva.com

Add a hook to register our app as the default handler for https://neeva.com and related URLs.  The associated `assetlinks.json` file on the server is already set up for this:
https://neeva.com/.well-known/assetlinks.json

Also bump the version number

## Checklists

### Pointers to help with review
* Figma link (if this is a UI change)
* Link to where the change is discussed (e.g. meeting notes, design docs)

### How was this tested?

### Manual test cases
1. Set Chrome as the default browser on the device
2. Fire an intent to https://neeva.com using adb: `adb shell am start -a "android.intent.action.VIEW" -d https://neeva.com`
3. Confirm that the Neeva app opens
